actions/AuthAction.js

```diff
@@ -74,8 +75,8 @@ function asAuthAction(action) {
     const permAuthCode = params.IMS_SERVICE_PERM_AUTH_CODE;
     const productContext = params.IMS_PRODUCT_CONTEXT;
 
-    // Extract the token from the params
-    const { imsOrg, accessToken } = params;
+    const accessToken = getAccessToken(params);
```

Comment:
In general, mixing authentication and payload data is considered a bad practice. It also doesn't work for `GET` requests. Therefore, I have implemented the use of the Authorization Bearer header and the `X-Org-Id` header.